docs/reference/objectmodel/preview-requirement-set/office.context.md

```diff
@@ -78,9 +78,9 @@ function write(message){
 Provides access to the properties for Office theme colors.
 
 > [!NOTE]
-> This member is not supported in Outlook on iOS or Android.
+> This member is only supported in Outlook on Windows.
```

Comment:
Subtle change, but better readability. Nice!

Reply:
And I think Mac is not supported either so definitely more accurate. Thx!